client/ayon_openrv/plugins/load/openrv/load_frames.py

```diff
@@ -97,22 +101,33 @@ def _get_sequence_range(self, context):
 
         """
         repre_entity = context["representation"]
+        version = context.get("version")
```

Comment:
Please use `_entity` suffix and the key is 100% there if there is `"representation"`.
```suggestion
        version_entity = context["version"]
```